sdk/core/Azure.Core.Amqp/src/AmqpAnnotatedMessage.cs

```diff
@@ -153,5 +155,27 @@ section switch
                 AmqpMessageSection.Footer => (_footer != null),
                 _ => throw new ArgumentException($"Unknown AMQP message section: { section }.", nameof(section))
             };
+
+        /// <summary>
+        /// Converts the AMQP message into its binary serialized form.
```

Comment:
Should we mention that this is the actual AMQP format described by the spec and not something proprietary?